05-build-dist.sh

```diff
@@ -2,6 +2,8 @@
 
 set -euo pipefail
 
+source config.sh
+
 cd rust
 mkdir -p build/riscv32em-unknown-none-elf/compiler-doc
-python x.py dist
+./x dist --target $TOOLCHAIN_HOST_TRIPLET --target riscv32em-unknown-none-elf
```

Comment:
Does the first target flag compile the dist for the specified target? Is this correct? Could I then use this flag to compile multiple dists, one for linux and one for macOS (maybe even one for windows) on the same machine?

Is the second flag instead specified because you removed it from the config.toml? If yes, why did you move it here?

Reply:
> In practice you usually cannot (easily) compile the toolchain so that it runs on a different system than the one on which you are compiling.

Ok, perfect, thanks!

> Yes. It was moved so that one of the target triplets can be set to be the same as the host system.

Ohh, I see now. @athei modified the `config.toml` file by removing the targets and making it dynamic using `config.sh`. Is my understanding correct? I want to make sure I understand the changes properly before I start experimenting with macOS distribution creation.

Reply:
Yes. Exactly. Before it was hardcoded to a linux target which would not be of much use when compiling on a macOS machine.